vendor/bat-native-ads/src/bat/ads/internal/account/account.cc

```diff
@@ -126,15 +126,9 @@ void Account::Deposit(const std::string& creative_instance_id,
   }
 
   deposit->GetValue(
-      creative_instance_id, [=](const bool success, const double value) {
```

Comment:
`[=]` is equal to capturing `this` by reference, so `base::Unretained(this)` is similar to the original implementation.